lib/modules/archivist/usercommands/rawSet.js

```diff
@@ -1,50 +1,23 @@
-exports.acl = ['*'];
-
-exports.execute = function (state, topic, index, data, mediaType, encoding, expirationTime, cb) {
-	var clientVault = state.archivist.getPrivateVault('client');
-	if (!clientVault) {
-		return state.error(null, 'Client vault not configured in writeOrder.', cb);
-	}
+const ArchivistModuleError = require('../ArchivistModuleError');
+const archivistModule = require('../');
 
-	var topicApi = state.archivist.getTopicApi(topic, clientVault.name);
-	if (!topicApi) {
-		return state.error(
-			null, 'Unable to load topic API on client vault for topic: ' + topic, cb
-		);
-	}
+exports.acl = ['*'];
 
-	var options = {
+exports.execute = async function (state, topic, index, data, mediaType, encoding, expirationTime) {
```

Comment:
Will `async` not break on Node 6?

Reply:
Yes, it would. It takes 7 with a flags and 7.6 and up without flag. 

I'll fix that.

Reply:
Going back to Promises-based doesn't really help us in terms of readability here. I'd rather wait for this to get merged once we remove support for Node <8.